Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string userFirstName = GetStringInput(message: "Enter your first name: ");
+            string userSurname = GetStringInput(message: "Enter your surname: ");
+            int userAge = GetIntInput(message: "Enter your age: ");
+            float userHeight = GetFloatInput(message: "Enter your height in cm: ");
+            float userWeight = GetFloatInput(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            if (weight <= 0 && height <=0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight + ".");
+                Console.WriteLine("Height cannot be less than zero, but was " + height + ".");
+            }
+            else if (weight <=0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight + ".");
+            }
+            else if (height <=0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Height cannot be less than zero, but was " + height + ".");
```

Comment:
Here you must print "equal or" too. Look at the Readme.md for detailed information.

Reply:
Hello. Thank you for the feedback. However, I am still very confused when you say I must print "equal or" in the line of code that writes "Height cannot be less than zero......".

The homework requirements for step 4 are here. I am following the bold message regarding height.
4) During BMI calculation, height or weight less or equal to 0 should print error message in console:
  - It should start with "Failed calculating BMI. Reason:" followed by a new line:
    **- "Height cannot be less than zero, but was X.", where X is the console input. (X <= 0)**
    - "Weight cannot be equal or less than zero, but was X.", where X is the console input (X <= 0)
    - If both height and weight are invalid, both validation messages should be printed to console (each in new line)

To me, what I already have coded is correct. Please clarify your comments. Thanks.

Reply:
[Screenshot_20200128_081817](https://user-images.githubusercontent.com/12687948/73243018-d14bbe00-41a6-11ea-96ae-5c530b95faf5.jpg)

This is the corrected version of the task description. You are right that in the Discord channel it is different. When in doubt always do what the test results tell you to do, because they do matter in the end. :) 

Sorry for the confusion!